app/assets/stylesheets/colours/_palette.scss

```diff
@@ -1,6 +1,272 @@
-// From https://github.com/alphagov/govuk_frontend_toolkit/blob/03204449dcbf28c2d48e7f6dd217f6abe9b3a518/stylesheets/colours/_palette.scss
+// =========================
+// updated 15 Oct 2019 14:14 BST
+// Palette change
+// https://designnotes.blog.gov.uk/2019/07/29/weve-updated-the-gov-uk-colours-and-font/
+// =========================
 
 $blue: #005ea5;
 $grey-1: #6f777b;
 $grey-2: #bfc1c3;
-$border-colour: $grey-2;
\ No newline at end of file
+$border-colour: $grey-2;
+$error-colour: #d4351c;
+$govuk-text-colour: #0b0c0c;
+
+// #DEE0E2 to #f3f2f1
+.button--secondary,
```

Comment:
This is now a duplicate of what is found in the 'govuk-override' scss file in the project? Or could be if referencing the estates services.

Reply:
Updated